test.py

```diff
@@ -0,0 +1,278 @@
+from __future__ import print_function
```

Comment:
This file will be removed :)  Though there is an implementation using slots here if that's desired :)